content/documentation/pages/1-installation/2-cloudfoundry/1-cf-cli.md

```diff
@@ -50,7 +50,7 @@ You can use `cf marketplace` to discover which plans are available to you, depen
 | concurrently connect to the same database instance without running
 | into connection limits. This usually means you can't use any free plans.
 
-## Install on Cloud Foundry
+## Manifest based installation on Cloud Foundry
```

Comment:
In this section, can we use variables here so that the user can just do a copy paste?

Reply:
discussed it with sabby, felt it was making it too noisy and using manifests is more prevalent in practice. ATM users have to cut-n-paste into a manifest.yaml file.  Can link to a sample one in our github I guess.  Will leave it as is for now.